maintainers/scripts/update.nix

```
@@ -1,7 +1,7 @@
 /*
-  To run:
+  Run with:
 
-      nix-shell maintainers/scripts/update.nix
+      maintainers/scripts/update
```

Shouldn't this be written as

```suggestion
      ./scripts/update
```

or something like that?

---

This line takes care of that

> ```
> cd "$(dirname "${BASH_SOURCE[0]}")/../.."
> ```

https://github.com/NixOS/nixpkgs/pull/283229/files#diff-6f027de646163b2d7d401a1d09c112d4fd73660577c466c1ab3c8dcf0574e79dR2